frontend/src/styles/global.css

```
@@ -1385,4 +1397,102 @@ input[aria-invalid="true"], select[aria-invalid="true"] { border-color: var(--da
+  font-family: var(--font-cond); font-weight: 700; font-size: 0.95rem; color: #fff; background: var(--amber-deep);
+}
+.emp-name-block { flex: 1; min-width: 0; }
+.emp-name { font-weight: 700; font-size: 1rem; color: var(--ink); white-space: nowrap; overflow: hidden; text-overflow: ellipsis; }
+.emp-role { font-size: 0.8rem; color: var(--ink-faint); margin-top: 2px; }
+.emp-status-pill {
+  display: inline-flex; align-items: center; gap: 6px; font-size: 0.66rem; font-family: var(--font-cond);
+  font-weight: 700; text-transform: uppercase; letter-spacing: 0.05em; flex-shrink: 0; padding: 5px 9px; border-radius: 999px;
+}
+.emp-status-pill.is-on { background: rgba(63, 178, 127, 0.14); color: var(--ok); }
+.emp-status-pill.is-off { background: var(--bg-press); color: var(--ink-faint); }
+.emp-status-dot { width: 6px; height: 6px; border-radius: 50%; background: currentColor; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)